### ISSUE TITLE
isDaylight() function

### ISSUE THREAD

**Issue by jonathancallahan** [MazamaScience/MazamaSatelliteUtils#17]
We should have a function which accepts a POSIXct and determines whether it is during the daylight over at least part of CONUS.

I would put this function inside a utils-time.R file where we may have other time related utility functions

Function signature:

* datetime = NULL -- any type of date and time (see goesaodc_listFiles.R for example parsing)
* region = "CONUS" -- a named region

For region = "CONUS", we should take the CONUS bounding box and choose locations in the middle of the east and west edges.

Look at the PWFSLSmoke::timeInfo() source code to see how to obtain the sunrise and sunset times from the maptools package.

Basically, we want to return TRUE if:

datetime  > sunrise on the east coast && datetime < sunset on the west coast
